backend/routes/filter-routes.js

```diff
@@ -4,16 +4,41 @@ const express = require('express');
 const routes = express.Router()
 
 const {PrismaClient, Prisma}= require('@prisma/client');
+const { NotFoundError } = require('../middleware/CustomErrors');
 const prisma = new PrismaClient();
 
 
 routes.get('/', async (req, res) => {
     return res.json({message: 'Hello World, the filter route works'});
 });
 
-routes.get('/:filter', async (req, res) => {
+
+routes.get('/:filter', async (req, res, next) => {
     const filter = req.params.filter;
-    return res.json({message: `Hello World, the filter ${filter} works`});
+    let queryResults = {}
+
+    if (filter == 'recent') {
```

Comment:
For these strings that we are checking for like "recent", "celebration", "inspiration" etc, I would use a constant.
```
enum Filters {
  RECENT = "recent"
  CELEBRATION = "celebration"
  INSPIRATION = "inspiration"
 ...
}
// use it like
else if (filter == Filters.INSPIRATION) 
...
```
 If this is used across multiple files, we can use a utils file. This way, we can avoid typos, like accidentally typing "receent" in a place and the check not working. With  the current structure, if the "recent" filter was changed to "recents", we would need to go and change the string everywhere, but if we used a constant, we would only need to change it in one place, everywhere else would be able to reference the constant with a variable.